### ISSUE TITLE
Help with workout page

### ISSUE THREAD

**Issue by ryan-peirce** [FitFast/FitFast-Android#52]
I don't know how to add categories from the database for the different workouts without just manually coding them in as strings in the workout activity. I either need a list of workout types and sub-types that are in the database (we may already have this) so i can hard code them in, or help writing the code to pull them from the database and populating the drop down menus with them. I'll also need help calling the fitness library to get the calculations i need.I'll be available to skype for a short time this weekend for this.

**Comment by scensorECHO** [comment]
Talk to Jon about which type of workout is what. 

There should be only 3 types, so you can hardcode it into strings.xml for now

**Comment by ryan-peirce** [comment]
So, we're good with hard coding the individual workouts as well? It's fine by me, since we don't have scaling as an issue to worry about.

**Comment by scensorECHO** [comment]
Yeah that's fine with me, the scope of the number of workouts doesn't need to scale significantly from the current standing of the fitness library. Confirm the types with Jon for the dropdown list and we'll move forward with it.

**Comment by jccalver** [comment]
Ryan here is the file that you are looking for
[WorkoutBreakDown.txt](https://github.com/FitFast/FitFast-Android/files/34385/WorkoutBreakDown.txt)

**Comment by ryan-peirce** [comment]
Thanks, i just committed what i have done. In each new activity, i have a "calcBurn" method. It is in this method that i have everything needed to call your methods from the fitness library. If you can take a look and add calls to your code, the workout page is done. We may need to skype for this so we both know everything is working on both ends. Not sure where the database comes into play either so Tom should probably take a look as well.
